tests/filecheck/dialects/qft/ops.mlir

```diff
@@ -0,0 +1,5 @@
+// RUN: QUOPT_ROUNDTRIP
+
+"test.op"() {gate = #qft.n<5>} : () -> ()
+
+// CHECK: "test.op"() {gate = #qft.n<5>} : () -> ()
```

Comment:
Could have a test with a use of the gate to check that number of qubits is working correctly